src/server/routes/api.ts

```diff
@@ -172,47 +159,35 @@ connection.once('open', () => {
                     break;
             }
         });
-        await allTissues.sort();
-        await allModels.sort();
 
         //////////////////////////////////////////////////////////////////////////////////////////
     });
 
-    GenesInfo.find({ nominations: { $gt: 0 } })
+    GenesInfo.find({ nominations: { $gt: 0 } }).lean()
         .sort({ hgnc_symbol: 1, tissue: 1, model: 1 })
         .exec(async (err, genes: GeneInfo[], next) => {
         if (err) {
             next(err);
         } else {
             tableGenesById = genes.slice();
-            // Table genes ordered by nominations
-            tableGenesByNom = await genes.slice().sort((a, b) => {
-                return (a.nominations > b.nominations) ? 1 :
-                    ((b.nominations > a.nominations) ? -1 : 0);
-            });
-
```

Comment:
Assuming this sort isn't needed anymore

Reply:
`tableGenesByNom` wasn't used anywhere, so I just removed it entirely.